neuralmonkey/nn/projection.py

```diff
@@ -19,9 +19,12 @@ def linear(inputs, size, scope="LinearProjection"):
         A tensor of shape batch x size
     """
     with tf.variable_scope(scope):
-        return tf.nn.seq2seq.linear(inputs, size, True)
+        # TODO this will be soon deprecated, should be rewrited with
+        # tf.contrib.layers.fully_connected and a support for list of tensors
```

Comment:
Podle mě je jedno, co tam bude za default, protože hodně záleží na aplikaci, co je lepší. _ReLU_ bezpochyby dává lepší výsledky u mnohovrstvého klasifikátru. _tanh_ je zase vhodný tam, kde je potřeba jenom jedna nelinearita nebo je to součástí nějakého složitějšího mechanismu, třeba uvnitř attentionu.